Sources/Fuzzilli/Lifting/JavaScriptLifter.swift

```diff
@@ -2268,4 +2304,89 @@ public class JavaScriptLifter: Lifter {
             fields[fields.count - 1] += body + "}"
         }
     }
+
+    // Every possible position of label
+    struct LabelPin {
+        var beginPos: Int
+        var hasLabel: Bool
+    }
+
+    enum LabelType {
+        case loopblock
+        case ifblock
```

Comment:
I guess for this PR, only `loopBlock` would be supported? The others should probably move to the corresponding PR